ffs/terms.md

```diff
@@ -0,0 +1,58 @@
+# Forum Funding System Terms & Conditions
+
+## Prelude Disclaimer (a note about the community)
+
+The community of Monero is really a very welcoming and friendly place. While these terms may seem strict to some, we just want to ensure that we have an environment that is fair, encourages innovation, protects the money of contributors, and protects contractors. In other words, we want a smooth FFS experience for everyone, contractors and contributors alike.
+
+We all want Monero to succeed, and we've got an awesome community that can make it happen. Let's just treat each other well, be honest with each other about anything unexpected that may arise, and make reasonable arguments for or against something, and, you know, be adults. In doing so, we shouldn't have any issues.
+
+## Introduction
+This document is written in order to clearly outline the expectations of the Monero community of a FFS proposal and the individual or team (hereafter 'contractor') that will fulfill it. By submitting a proposal, you agree that you have read and understood these terms. Failure to comply with these terms will lead to an investigation by the Monero Core Team (as outlined [here](https://camo.githubusercontent.com/757dcb14433d90308a454d84ed1cd5f7a63403a1/68747470733a2f2f6765746d6f6e65726f2e6f72672f626c6f672f6173736574732f323031352d796561722d696e2d7265766965772f676f7665726e616e63652d70726f636573732e6a7067) and expanded on below)  which may lead to the dismissal of contractor(s). These terms are not legally binding: Monero is not a corporation but, if you do not follow the terms, there will be repercussions regarding your contract or receiving payment.
+
+Whenever it is mentioned that something will be decided by 'the community' it is understood that this will mean during one of the weekly meetings (dev or community, whichever is more appropriate). The item must be added to the GitHub agenda/issue ASAP, and if the meeting is less than a week away then the item must be postposned to the next meeting. This is so anyone who wishes to attend the meeting to vote and make their voice heard is given ample opportunity and notice to do so.
+
+If a community member is unnable to attend a meeting but would like to vote on an issue, he/she may do so by proxy by providing permission in writing to the community, specifically naming the individual who may vote on his/her behalf. In the event that a community member does not attend a meeting and does not like the decision that was agreed upon during said meeting, tough. He/she should have been there or voted by proxy.
+
+All votes will be done by a show of hands (o/) for 'yays' and 'nays', the votes will be tallied, the number verified, and the majority shall win the vote. We do not have an electoral college.
```

Comment:
>All votes will be done by a show of hands (o/) for 'yays' and 'nays', the votes will be tallied, the number verified, and the majority shall win the vote. We do not have an electoral college.

This looks dangerous and ineffective. It's incredibly cheap to create identities, come to a meeting, vote, and disappear.

As for solutions, off the top of my head, [Decred](https://www.decred.org/) has applied their solution to this problem very well, IMHO.

Reply:
Agree with @anonimal. [Unique Ring Signatures](https://github.com/monero-project/urs) could also be an alternative solution, but not sure what "membership" criteria should be used to create a key ring.

Reply:
> Decred has applied their solution to this problem very well,

Their voting system is very interesting (explained [here](https://docs.decred.org/getting-started/user-guides/agenda-voting/)). Should we start thinking to integrate something similar? More Monero expands, more urgent this gets